### PR TITLE
Winpack build using VC 16 for 4.x branch

### PR DESCRIPTION
**Pull request by asenyaev**

### CODE REVIEW

File: .github/workflows/OCV-WinPack-4.x-W10.yaml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    - name: Clean
+      timeout-minutes: 60
+      run: cd ${{ github.workspace }} && rm -rf *
+    - name: Fetch opencv
+      timeout-minutes: 60
+      run: cd ${{ github.workspace }} && git clone --branch ${{ env.BRANCH_NAME }} --reference %GIT_CACHE%\opencv.git git@github.com:opencv/opencv.git
+    - name: Cmake OpenCV
+      timeout-minutes: 60
+      env:
+        EXTRA_CMAKE_OPTIONS: '-DWITH_OBSENSOR=OFF -DOPENCV_PYTHON_INSTALL_PATH=python -DCMAKE_INSTALL_PREFIX=${{ github.workspace }}/install -DBUILD_SHARED_LIBS=ON -DBUILD_PERF_TESTS=OFF -DBUILD_TESTS=OFF -DBUILD_EXAMPLES=OFF -DINSTALL_PDB=ON -DINSTALL_PDB_COMPONENT_EXCLUDE_FROM_ALL=OFF -DCPU_BASELINE=SSE3 -DOPENCV_INSTALL_DATA_DIR_RELATIVE=${{ github.workspace }}\\opencv -DWITH_LAPACK=OFF -DVIDEOIO_PLUGIN_LIST=all -DINSTALL_CREATE_DISTRIB=ON -DBUILD_DOCS=OFF -DOPENCV_GENERATE_SETUPVARS=ON -DWITH_OPENCL=ON -DBUILD_opencv_python3=OFF -DBUILD_opencv_python2=OFF -DWITH_TBB=OFF -DWITH_CUDA=OFF -DBUILD_opencv_java=OFF'
```
Comment (asenyaev) [inline .github/workflows/OCV-WinPack-4.x-W10.yaml:32]: `-DWITH_OBSENSOR=OFF` is set temporary, while https://github.com/opencv/opencv/pull/22327 isn't merged.

**Review by asmorkalov — APPROVED** [review]
:+1: LGTM!

**Comment by asenyaev** [comment]
Need to update, because some of the components are missing:
- Python 2.7, Python 3.5, Python 3.6, Python 3.7, Python 3.8, Python 3.9
- Java x86

**Comment by asmorkalov** [comment]
Let's discuss python 2.7 support on Friday. Most probably we can safely drop it.

File: .github/workflows/OCV-WinPack-4.x-W10.yaml (hunk 1 above)
Comment (fengyuentau) [reply .github/workflows/OCV-WinPack-4.x-W10.yaml:32]: This pr has been merged.

**Comment by fengyuentau** [comment]
Does 'WinPack' stand for the release package for Windows platform like `opencv-4.6.0-vc14_vc15.exe` attached in https://github.com/opencv/opencv/releases/tag/4.6.0?

**Comment by asmorkalov** [comment]
Yes

**Review by asmorkalov — APPROVED** [review]

**Review by asmorkalov — APPROVED** [review]
:+1:

**Review by asmorkalov — APPROVED** [review]
:+1:

**Pull request merged**